src/js/view/view.js

```diff
@@ -391,7 +391,7 @@ define([
                 '.jw-option.jw-active-option',
                 // slider fill color
                 '.jw-progress'
-            ], 'background', activeColor);
+            ], 'background', 'none ' + activeColor);
```

Comment:
@pajong in Safari, it's necessary to clear the background image so that it does not affect custom colors or gradients.